README.md

```diff
@@ -22,6 +23,15 @@ After you've made some changes, kill (Ctrl+C) the existing `provider-ceph` and r
 make run
 ```
 
+### Debugging
+Spin up the test environment, but with `provider-ceph` running locally in your terminal:
+
+```
+make mirrord.cluster mirrord.run
```

Comment:
This resulted in the following for me, any ideas?
```
x preparing to launch process
      ✓ layer extracted                                                                                                            Error:   × Create agent failed. KubeError(
  │     Api(
  │         ErrorResponse {
  │             status: "Failure",
  │             message: "deployments.apps \"provider-ceph-provider-cep\" not found",
  │             reason: "NotFound",
  │             code: 404,
  │         },
  │     ),
  │ )
  ```

Reply:
probably a _me_ problem.. I haven't run a "local" dev environment in some time so I'm a bit out of practice debugging